src/phase2.hpp

```diff
@@ -149,100 +157,100 @@ std::vector<uint64_t> RunPhase2(
             0,
             strategy_t::quicksort);
 
+        if (table_index == 7) {
+            // we don't use the sort manager in this case, so we can use the
+            // memort as a write buffer instead
```

Comment:
`memort`

Reply:
same thing here. It's supposed to say "memory", but this comment is removed in a later commit, when the memory is heap allocated instead.